recipe/meta.yaml

```
@@ -1,43 +1,86 @@
 {% set name = "mysqlclient" %}
-{% set version = "2.0.3" %}
+{% set version = "2.2.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f6ebea7c008f155baeefe16c56cd3ee6239f7a5a9ae42396c2f1860f08a7c432
+  url: https://github.com/PyMySQL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 518f256556b518bd947cdd682b4d7c2263427588296e77cc1f30e5b95f5be13f
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  # Skipping Windows due to build challenges, reliance on mariadbclient which is not available
+  skip: true  # [py<38 or win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - pkg-config
   host:
     - python
     - pip
-    - mysql-connector-c
+    - mysql-devel
```

```suggestion
    - mysql-devel 9.3
```
Pin to ensure ABI compatibility